packages/office-addin-manifest/src/manifestHandler/getManifestHandler.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { ManifestHandler } from "./manifestHandler";
+import { ManifestHandlerJson } from "./manifestHandlerJson";
+import { ManifestHandlerXml } from "./manifestHandlerXml";
+
+export async function getManifestHandler(manifestPath: string): Promise<ManifestHandler> {
```

Comment:
Couldn't this be part of manifestOperations (the only places it's used . . . beside testing, which only really needs to happen because it's being exported which it wouldn't need to be).